addons/rating/models/mail_thread.py

```diff
@@ -166,8 +166,6 @@ def rating_apply(self, rate, token=None, rating=None, feedback=None,
     def message_post(self, **kwargs):
         rating_id = kwargs.pop('rating_id', False)
         rating_value = kwargs.pop('rating_value', False)
-        # create rating.rating record linked to given rating_value. Using sudo as portal users may have
-        # rights to create messages and therefore ratings (security should be checked beforehand)
```

Comment:
Note: this sudo/comment made no sense, as `message_post` is a public method there is no such things as "checking beforehand".

Reply:
When creating the message, it checks access on related record. So the wording is a bit misleading but yes checks are done.